tidb-lightning/tidb-lightning-distributed-import.md

```diff
@@ -16,11 +16,19 @@ You can use TiDB Lightning to import data in parallel in the following scenarios
 
 > **Note:**
 >
+<<<<<<< HEAD
 > Parallel import only supports the initialized empty tables in TiDB. It does not support migrating data to tables with data written by existing services. Otherwise, data inconsistencies may occur.
+=======
+> - Parallel import only supports initialized empty tables in TiDB and does not support migrating data to tables with data written by existing services. Otherwise, data inconsistencies may occur.
+>
+> - Parallel import is usually used in the physical import mode. You need to configure `incremental-import = true`.
+>
+> - Apply only one backend at a time when using multiple TiDB Lightning instances to import data to the same target. For example, you cannot import data to the same TiDB cluster in both the physical and logical import modes at the same time.
+>>>>>>> fe05a21fe (lightning: update distributed import (#13483))
```

Comment:
```suggestion
> - Parallel import only supports initialized empty tables in TiDB and does not support migrating data to tables with data written by existing services. Otherwise, data inconsistencies may occur.
>
> - Parallel import is usually used in the physical import mode. You need to configure `incremental-import = true`.
>
> - Apply only one backend at a time when using multiple TiDB Lightning instances to import data to the same target. For example, you cannot import data to the same TiDB cluster in both the physical and logical import modes at the same time.
```